src/pages/dev/general-message-passing/gas-services/intro.mdx

```diff
@@ -1,17 +1,43 @@
+The Axelar network routes every [General Message Passing (GMP)](https://docs.axelar.dev/dev/general-message-passing/overview) transaction to its destination chain. The final step of this process involves the application paying the gas fees for the transaction. This can be done in one of two ways:
 
-## Gas Receiver
-For any General Message Passing (GMP) transaction, the Axelar network routes transactions to their destination chains. Execution is the final step of the pipeline to the specified destination contract address on the destination chain. It is invoked in one of two ways.
+1. **On the destination chain:** Manually paid on the destination chain
+1. **On the source chain:** Prepaid to the `AxelarGasService` contract on the source chain, thus allowing Axelar to execute the final transaction on the destination chain
 
-1. Manually paid by the user/application on the destination chain.
-2. Executed automatically by Axelar if the user/application prepays gas to the Gas Receiver contract on the source chain.
+Typically, gas fees are manually paid on the destination chain, but Axelar’s gas services provide the ability to prepay the full estimated cost for any GMP transaction with a single payment on the source chain. Axelar’s relay services manage the full pipeline. Once gas is paid to the Gas Receiver contract, the relayer services pick up the payment and automatically execute the final GMP call on the destination chain.
 
-Gas Receiver is a smart contract deployed on every EVM that is provided by Axelar. Our gas services provide users/applications the ability to prepay the full estimated cost for any GMP transaction (from source to destination chains) with the convenience of a single payment on the source chain, relying on Axelar's relay services to manage the full pipeline. Once gas is paid to Gas Receiver for a GMP transaction, Axelar's relayer services pick up the payment and automatically execute the final General Message Passing call.
+## Increase gas
 
-Developers can use Axelar gas services by prepaying upfront the relayer gas fee on the source chain, thereby covering the cost of gas to execute the final transaction on the destination chain.
\ No newline at end of file
+If the amount of prepaid gas is not enough — for example, if the destination chain is congested with transfers, causing a gas fee increase for each transaction — the application can resubmit a new amount of gas to pay for the transaction.
+
+## The Executor Service
```

Comment:
We should simplify and only talk about the "Relay Service" "Executor Service" or "Gas Service". I prefer gas service.

Reply:
That sounds good to me. I think we should default to "Gas Service" as well (proper noun). Will update the style guide accordingly.